### PR TITLE
Have all images configured in one place

### PR DESCRIPTION
**Pull request by coulof**
For offline installation, it isn't straightforward to modify the _helpers.tpl which is not needed anyways.

One variable with all images makes it much more manageable.

Uncomment the podmon arguments.

Please comment if there are impacts outside the helm chart (in the offline bundle ? QA ? or elsewhere ?)
# Description
A few sentences describing the overall goals of the pull request's commits.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| JIRA KRV-12738 |

# Checklist:

- [x] Have you run format,vet & lint checks against your submission?
- [ ] Have you made sure that the code compiles?
- [ ] Did you run the unit & integration tests successfully?
- [ ] Have you maintained at least 90% code coverage?
- [ ] Have you commented your code, particularly in hard-to-understand areas
- [ ] Have you done corresponding changes to the documentation
- [ ] Did you run tests in a real Kubernetes cluster?
- [ ] Backward compatibility is not broken

# How Has This Been Tested?
Tested on my local cluster

Will be used at Intel

### CODE REVIEW

File: helm/csi-powermax/values.yaml [hunk 1]
```diff
@@ -53,7 +53,25 @@ version: "v2.7.0"
...
+  attacher: registry.k8s.io/sig-storage/csi-attacher:v4.3.0
+  provisioner: registry.k8s.io/sig-storage/csi-provisioner:v3.5.0
+  snapshotter: registry.k8s.io/sig-storage/csi-snapshotter:v6.2.2
+  resizer: registry.k8s.io/sig-storage/csi-resizer:v1.8.0
+  registrar: registry.k8s.io/sig-storage/csi-node-driver-registrar:v2.8.0
+  healthmonitor: registry.k8s.io/sig-storage/csi-external-health-monitor-controller:v0.9.0
+  # CSM sidecars
+  replication: dellemc/dell-csi-replicator:v1.5.0
+  vgsnapshotter: dellemc/csi-volumegroup-snapshotter:v1.2.0
+  podmon: dellemc/podmon:v1.6.0
```
Comment (boyamurthy) [inline helm/csi-powermax/values.yaml:68]: vgsnapshotter and podmon can be removed for powermax since support is not yet available.

**Review by boyamurthy — COMMENTED** [review]
LGTM.

**Review by shanmydell — APPROVED** [review]

**Review by boyamurthy — APPROVED** [review]
LGTM.

**Comment by shanmydell** [comment]
Hi @coulof,
Changes look good, Make standalone helm chart changes have been merged now, and we request that you make these changes against the centralized repository.

-Yamuna

**Pull request closed without merging**